core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -1159,6 +1159,36 @@ protected boolean isUnparserTest() {
         .ok("SELECT ((`TBL`.`FOO`(0).`COL`).`BAR`)\nFROM `TBL`");
   }
 
+  @Test public void testRowValueExpression() {
+    String expected = "INSERT INTO \"EMPS\"\n"
+            + "VALUES (ROW(1, 'Fred')),\n"
+            + "(ROW(2, 'Eric'))";
+    sql("insert into emps values ROW(1,'Fred'),ROW(2, 'Eric')")
+            .withDialect(SqlDialect.DatabaseProduct.CALCITE.getDialect())
+            .ok(expected);
+
+    expected = "INSERT INTO `emps`\n"
+            + "VALUES (1, 'Fred'),\n"
+            + "(2, 'Eric')";
+    sql("insert into emps values (1,'Fred'),(2, 'Eric')")
```

Comment:
I think you should use the same text in sql(), and check both for 
`sql("insert into emps values ROW(1,'Fred'),ROW(2, 'Eric')")`
and
`sql("insert into emps values (1,'Fred'),(2, 'Eric')")`

Reply:
sorry,I didn't notice that
Write the wrong